yt_dlp/extractor/youtube/__init__.py

```diff
@@ -0,0 +1,20 @@
+# flake8: noqa: F401
+from ._clip import YoutubeClipIE
+from ._mistakes import YoutubeTruncatedIDIE, YoutubeTruncatedURLIE
+from ._notifications import YoutubeNotificationsIE
+from ._redirect import (
+    YoutubeConsentRedirectIE,
+    YoutubeFavouritesIE,
+    YoutubeFeedsInfoExtractor,
+    YoutubeHistoryIE,
+    YoutubeLivestreamEmbedIE,
+    YoutubeRecommendedIE,
+    YoutubeShortsAudioPivotIE,
+    YoutubeSubscriptionsIE,
+    YoutubeWatchLaterIE,
+    YoutubeYtBeIE,
+    YoutubeYtUserIE,
+)
+from ._search import YoutubeMusicSearchURLIE, YoutubeSearchDateIE, YoutubeSearchIE, YoutubeSearchURLIE
+from ._tab import YoutubePlaylistIE, YoutubeTabBaseInfoExtractor, YoutubeTabIE
+from ._video import YoutubeBaseInfoExtractor, YoutubeIE
```

Comment:
do we need to import `YoutubeBaseInfoExtractor` from `_video` here, or can we import from `_base`?

and should `YoutubeTabBaseInfoExtractor` go in `_base` instead of `_tab`?

Reply:
Whoops, good catch

> and should YoutubeTabBaseInfoExtractor go in _base instead of _tab?

I was originally thinking having a `_tabbase`, but it's tightly coupled to tab so I've put them both into the same module.